openfl-tutorials/experimental/Keras_MNIST_with_CPU.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "4dbb89b6",
+   "metadata": {},
+   "source": [
+    "Install pre-requisites and intialize data, model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f7f98600",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Install openfl and required packages for the workflow APIs to function\n",
+    "%pip install git+https://github.com/securefederatedai/openfl.git\n",
+    "%pip install -r workflow_interface_requirements.txt\n",
+    "\n",
+    "#Install Tensorflow and MNIST dataset if not installed\n",
+    "%pip install tensorflow==2.13\n",
+    "\n",
+    "# Uncomment this if running in Google Colab and set USERNAME if running in docker container.\n",
+    "# !pip install -r https://raw.githubusercontent.com/intel/openfl/develop/openfl-tutorials/experimental/workflow_interface_requirements.txt\n",
+    "# import os\n",
+    "# os.environ[\"USERNAME\"] = \"colab\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c2352436",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "import tensorflow.python.keras as keras\n",
+    "import matplotlib.pyplot as plt\n",
+    "from keras import backend as K\n",
```

Comment:
nit, some of the imports are unused and some are reimported.

Reply:
Removed redundant packages.